gems/jquery-rails/CVE-2020-11022.yml

```diff
@@ -0,0 +1,73 @@
+---
+gem: jquery-rails
+cve: 2020-11022
+ghsa: gxr4-xjj5-5px2
+url: https://github.com/jquery/jquery/security/advisories/GHSA-gxr4-xjj5-5px2
+title: Potential XSS vulnerability in jQuery
+date: 2020-04-29
+description: |
+  "### Impact\nPassing HTML from untrusted sources - even after sanitizing
+  it - to one of jQuery's DOM manipulation methods (i.e. `.html()`, `.append()`, and
+  others) may execute untrusted code.\n\n### Patches\nThis problem is patched in jQuery
+  3.5.0.\n\n### Workarounds\nTo workaround the issue without upgrading, adding the
+  following to your code:\n\n```js\njQuery.htmlPrefilter = function( html ) {\n\treturn
+  html;\n};\n```\n\nYou need to use at least jQuery 1.12/2.2 or newer to be able to
+  apply this workaround.\n\n### References\nhttps://blog.jquery.com/2020/04/10/jquery-3-5-0-released/\nhttps://jquery.com/upgrade-guide/3.5/\n\n###
+  For more information\nIf you have any questions or comments about this advisory,
+  search for a relevant issue in [the jQuery repo](https://github.com/jquery/jquery/issues).
+  If you don't find an answer, open a new issue."
```

Comment:
This appears to be a double-escaped String? Is this a bug in the github sync script?

Reply:
Corrected